docs/virtual_machines/network_binding_plugins.md

```diff
@@ -220,3 +220,168 @@ spec:
                 dhcp4: true
         name: cloudinitdisk
 ```
+
+## Passt network binding plugin
+
+[Plug A Simple Socket Transport](https://passt.top/passt/about/) is an enhanced
```

Comment:
Just wondering, if this is replacing the original passt impl. Does it replace the original doc as well?
If so, we need to copy its content here. e.g comparison between binding, restrictions, etc.